components/ai_chat/core/browser/engine/engine_consumer.h

```diff
@@ -77,8 +77,9 @@ class EngineConsumer {
 
   virtual void UpdateModelOptions(const mojom::ModelOptions& options) = 0;
 
-  void SetMaxPageContentLengthForTesting(int max_page_content_length) {
-    max_page_content_length_ = max_page_content_length;
+  void SetMaxAssociatedContentLengthForTesting(
+      int max_associated_content_length) {
```

Comment:
nit: make types consistent 
```suggestion
      uint32_t max_associated_content_length) {
```

Reply:
I didn't touch these types. Did you run tests to make sure nothing breaks? 😊